src/MusicStore.Spa/Infrastructure/SimpleMapper.cs

```diff
@@ -14,8 +14,12 @@ public class SimpleMapper
 
         public static TDest Map<TSource, TDest>(TSource source, TDest dest)
         {
+#if ASPNET50
+            return AutoMapper.Mapper.Map(source, dest);
+#else
             var map = (Func<TSource, TDest, TDest>)_mapCache.GetOrAdd(Tuple.Create(typeof(TSource), typeof(TDest)), _ => MakeMapMethod<TSource, TDest>());
             return map(source, dest);
+#endif
```

Comment:
@yishaigalatzer - Moved AutoMapper mapping to SimpleMapper Map method. Does this look better?